htsget-http-actix/src/main.rs

```diff
@@ -14,6 +15,9 @@ This executable doesn't use command line arguments, but there are some environme
 * HTSGET_IP: The ip to use. Default: 127.0.0.1
 * HTSGET_PORT: The port to use. Default: 8080
 * HTSGET_PATH: The path to the directory where the server should be started. Default: Actual directory
+* HTSGET_REGEX: The regular expression that should match an ID. Default: ".*"
+* HTSGET_REPLACEMENT: The replacement expression. Default: "$0"
+For more information about the regex options look in the documentation of the regex crate.
```

Comment:
I'd remove this reference to the regex crate if it doesn't have a link.

Reply:
I think I wanted to add the link later and then forgot about it 😥 I will add it